Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "traceme"
+version = "0.1.0"
+authors = ["Edd Barrett <vext01@gmail.com>"]
+
+[dependencies]
+libc = "0.2.33"
+clippy = {version = "*", optional = true}
```

Comment:
We don't want to add Clippy as a dependency. `cargo clippy` runs Clippy without the dependency.

Reply:
I was following the clippy instructions here:
https://github.com/rust-lang-nursery/rust-clippy#usage

`cargo clippy` only works if you have clippy installed, whereas the recommended way always works.

(Note that it's an optional dependency, so by default you won't get it.)

Reply:
Admittedly `cargo clippy` is easier, but I think we should leave these lines in for those that want to do it the other way.

Reply:
Let's not clog up the dependencies please (there's no advantage to doing so).

Reply:
I disagree, but as you wish.